release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -146,6 +146,10 @@ For more information, see xref:../updating/updating-cluster-prepare.adoc#updatin
 ==== Minimum required permissions for Microsoft Azure to install and delete an {product-title} cluster
 In {product-title} {product-version}, instead of using the built-in roles, you can now define your custom roles to include the minimum required permissions for Microsoft Azure to install and delete an {product-title} cluster. These permissions are available for installer-provisioned infrastructure and user-provisioned infrastructure.
 
+[id="ocp-4-13-updating-clusters-migrating-cluster-to-multi-arch"]
+==== Convenient single-architecture to multi-architecture payload migration
```

Comment:
```suggestion
==== Single-architecture to multi-architecture payload migration
```
"Convenient" is subjective. I suggest avoiding it.

Reply:
good call :) ty